src/main.ts

```diff
@@ -93,9 +119,24 @@ const command: Command = {
 		});
 	},
 	run(helper: Helper, args: TestArgs) {
+		/* istanbul ignore next */
+		function unhandledRejection(reason: any) {
+			console.log('Unhandled Promise Rejection: ');
+			console.log(reason);
+		}
+
+		process.on('unhandledRejection', unhandledRejection);
+
 		return new Promise((resolve, reject) => {
 			if (!helper.command.exists('build')) {
-				reject(Error('Required command: \'build\', does not exist. Have you run npm install @dojo/cli-build?'));
+				reject(Error(`Required command: 'build', does not exist. Have you run 'npm install ${CLI_BUILD_PACKAGE}'?`));
+			}
+			try {
+				const projectName = require(path.join(process.cwd(), './package.json')).name;
+				console.log(blue.bold(`\n▶ Building "${projectName}":`) + `\n`);
```

Comment:
I believe that dojo build, eject and create should all be using the same output format at the moment.